documentation/src/docs/metrics.md

```diff
@@ -22,13 +57,15 @@ public interface StatsProvider {
 }
 ```
 
-- start(): Initializes [MetricRegistry](http://metrics.dropwizard.io/3.1.0/manual/core/#metric-registries) and reporters for our Metrics service. 
-- close(): Shutdown of Metrics service.
-- createStatsLogger(): Creates and returns a StatsLogger instance, which is used to retrieve a metric and do metric insertion and collection in Pravega code. 
-- createDynamicLogger(): Create a dynamic logger.
+- `start()`: Initializes [MetricRegistry](http://metrics.dropwizard.io/3.1.0/manual/core/#metric-registries) and Reporters for our Metric service.
+- `close()`: Performs Shutting down of Metric Service.
+- `createStatsLogger()`: Creates and returns a `StatsLogger` instance, which is used to retrieve a metric and performs metric insertion and collection in Pravega code.
+- `createDynamicLogger()`: Creates a Dynamic Logger.
```

Comment:
@kevinhan88 could you suggest some other term instead ` Pravega code`.

Reply:
Sorry not quite getting the question.

Reply:
@kevinhan88 like can we rephrase as Pravega application instead Pravega code or some other relevant term.

Reply:
Got you now. How about this?
'createStatsLogger()': Create a StatsLogger instance which is used to register and return metric objects. Application code could then perform metric operations directly with the return metric objects.

Reply:
Accepted suggestion